cloud

```diff
@@ -1243,6 +1244,22 @@ function pubsub_withdraw() {
     pubsub_topic_withdraw "$project" "$pick_notifications_trigger_topic"
 }
 
+# Deploy a Google Cloud Storage Bucket
+# Args: project storage_bucket
+function storage_deploy() {
+    declare -r project="$1"; shift
+    declare -r bucket="$1"; shift
+    gsutil -q mb -p "$project" -c STANDARD -l "us-central1" -b on "gs://$bucket"
+    gsutil iam ch allUsers:objectViewer "gs://$bucket/"
```

Comment:
Um, you are still not setting the original `TMPDIR` here, as I suggested.

Reply:
When I was testing it with those code changes I didn't got those errors, but on further testing I got those errors again. Updated the code and tested to all works fine.

Reply:
So, will you add setting the `TMPDIR`?

Reply:
Yes